agent/artifact_uploader.go

```diff
@@ -83,7 +83,8 @@ func (a *ArtifactUploader) Collect() (artifacts []*api.Artifact, err error) {
 		// Resolve the globs (with * and ** in them)
 		files, err := zglob.Glob(globPath)
 		if err == os.ErrNotExist {
-			return nil, nil
+			logger.Info("No files matched glob: %s", globPath)
```

Comment:
Technically - `zglob` only returns `os.ErrNotExist` when you pass in a non-globbed path. So the message would look this `INFO - No files matched glob: some/path/with/no/globs`. Maybe you'd want a slight re-wording of the log?

`"File not found: %s"`